detection_rules/etc/non-ecs-schema.json

```diff
@@ -144,5 +144,9 @@
     "auditd.data.a1": "keyword",
     "auditd.data.a2": "keyword",
     "auditd.data.a3": "keyword"
+  },
+  "logs-aws.cloudtrail-*": {
+    "aws.cloudtrail.flattened.request_parameters.cidrIp": "keyword",
+    "aws.cloudtrail.flattened.request_parameters.fromPort": "keyword"
```

Comment:
any idea why these aren't populated in the integration?

Reply:
The request and response dictionaries have important context that should be parsed out in general. Ref -> https://github.com/elastic/integrations/issues/9586

I am hoping to work with integrations to figure this out:

1. Do we need to give a list of nested structures to pull out if they exist?
2. Can we dynamically parse these into their own fields?

Reply:
@brokensound77 - I do not anticipate that these changes to the integration will come any time soon, however, the issue is there for tracking as we can tune the rules when a decision and solution are determined. Any other thoughts before I resolve this?

Reply:
Nope. Sgtm. Thanks for the details 👍